modules/runtime/pages/reporting-app.adoc

```diff
@@ -0,0 +1,44 @@
+= Bonita Reporting App 
+
+:description: An application for analytics and reporting. 
+
+
+{description} +
+Bonita Reporting app will enable you to view statistics and to extract process execution reports with meaningful data from Bonita, for process performance analysis and for pinpointing areas of improvement.
+
+[NOTE]
+====
+
+*For Enterprise, Performance, Efficiency, and Teamwork editions only.*
+
+In keeping with Bonitasoft’s promise to continuously deliver valuable, high quality features, we had to relinquish the Analytics and Reports page starting from Bonita 2021.2.
+
+Bonita Reporting App, is a Bonita application compatible with Bonita 2021.2 and available for download on the Customer Portal/Customer Service Center.
+
+An Enterprise licence is required. 
+====
+
+== Overview 
+
+The bonita-dashboard-app-1.1.1.jar, contains two reports “Process Historical Data” and “Human tasks historical data”.  
+
+=== Process historical data
+
+For analyzing zases execution data of a chosen process in a defined period of time.
```

Comment:
What are the main use cases of this page / report? What can the user do / understand by reading this report?

Reply:
More over a screenshot of the graph and the explanations on the 2 axis could be very instructive to understand what the graph shows, what do you think?